src/Encamina.Enmarcha.Bot/Logging/ApplicationInsigthsConversationScopedLogger.cs

```diff
@@ -68,7 +68,7 @@ public bool IsEnabled(LogLevel logLevel)
     }
 
     /// <inheritdoc />
-    public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Exception exception, Func<TState, Exception, string> formatter)
+    public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Exception exception, Func<TState, Exception?, string> formatter)
```

Comment:
It does not make sense here for the `formatter` to receive a nullable exception. The `formatter` type should be as it was: `Func<TState, Exception, string>`. If the compiler is giving you an warning here, lets understand it first because the solution might be different and not as simpler as adding an `?` :/